api/sql/schema/750$rule.decision.lookup.sql

```diff
@@ -1,17 +1,17 @@
 ALTER PROCEDURE [rule].[decision.lookup]
-    @channelId BIGINT, -- the id of the channel triggering transaction
-    @operation varchar(100), -- the operation name
-    @operationDate datetime, -- the date when operation is triggered
-    @sourceAccount varchar(100), -- source account number
+    @channelId BIGINT, -- the id of the channel triggering TRANSACTION
+    @operation VARCHAR(100), -- the operation name
+    @operationDate datetime, -- the DATE WHEN operation IS triggered
```

Comment:
Is there a way to avoid changes in comments with the tool you are using? This will not look good in the documentation that is generated.

Reply:
Sorry, Not sure about it in vscode. Now I will remove the comment changes which looks ugly by manually and will take care of such thing in future.

Reply:
I have reverted the comment changes. Please check it